test/system/components/editor_test.rb

```diff
@@ -120,13 +135,62 @@ class EditorTest < ApplicationSystemTestCase
         sign_in!(user)
         visit edit_track_exercise_path(track, exercise)
 
+        # Make sure component is mounted
+        sleep 0.5
         click_on "Feedback"
 
-        assert_text "request mentoring"
+        # in JSX &nbsp; is used and this is thats equivalent here
```

Comment:
Maybe be explicit out what "this" is:

```suggestion
        # Use "\n" to match where JSX uses "&nbsp;"
```